resources/niri-session

```diff
@@ -1,4 +1,34 @@
-#!/bin/sh
+#!/usr/bin/env sh
+
+print() {
+    case $(basename $SHELL) in
+        fish|bash|zsh)
+            echo -e $@
+            ;;
+        *)
+            # Remove all \x1B escapes from the text.
+            echo $@ | sed 's@\\x1b\[[0-9;]*m@@gi';
+            ;;
+    esac
+}
```

Comment:
you could also just use `printf`? all of the shells you match for `echo -e` have a working printf (and many you don't, including dash/ash etc), and it even has a posix spec, so you can call it directly